src/scripts/main.js

```diff
@@ -1,17 +1,51 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.querySelector('body');
+  const div = document.createElement('div');
+
+  switch (type) {
+    case 'success' :
+      div.classList = 'notification success';
+      break;
+    case 'warning' :
+      div.classList = 'notification warning';
+      break;
+    case 'error' :
+      div.classList = 'notification error';
+      break;
+  }
```

Comment:
you dont need switch here.
```suggestion
div.classList.add('notification', type)
```